bedrock/products/templatetags/misc.py

```diff
@@ -80,13 +80,19 @@ def vpn_subscribe_link(ctx, entrypoint, link_text, plan=None, class_name=None, l
     if plan in ['12-month', '6-month', 'monthly']:
 
         # Set a default plan ID using page locale. This acts as a fallback should geo-location fail.
-        pricing_params = settings.VPN_VARIABLE_PRICING.get(lang, settings.VPN_VARIABLE_PRICING['us'])
-        plan_default = pricing_params[plan]['id']
+        cc = lang.split('-')[1].lower() if lang and '-' in lang else lang
```

Comment:
should the last part be `lang.lower()` or will the langs with no dash in them definitely be lower case?

Reply:
Worth adding just to be extra safe 👍